.github/workflows/ubuntu.yml

```diff
@@ -16,12 +16,13 @@ jobs:
           sudo apt install cmake git g++ pkg-config libgraphicsmagick++-dev
           sudo apt install libturbojpeg-dev libexif-dev libswscale-dev libpng-dev
           sudo apt install libavcodec-dev libavformat-dev libavdevice-dev
+          sudo apt install libopenslide-dev
           sudo apt install pandoc
       - name: Get the Source
         uses: actions/checkout@v2
       - name: Build timg
         run: |
           mkdir build
           cd build
-          cmake .. -DWITH_VIDEO_DECODING=On -DWITH_VIDEO_DEVICE=On
+          cmake .. -DWITH_VIDEO_DECODING=On -DWITH_VIDEO_DEVICE=On WITH_SLIDE_IMAGES=On
```

Comment:
this looks like a `-D` is missing ?

Reply:
Maybe in the macos yml file, you also want to be as explicit and add the WITH_SLIDE_IMAGES ?

Reply:
(solved)